src/dc_lot.rs

```diff
@@ -141,15 +142,12 @@ pub unsafe fn dc_lot_fill(
             (*lot).text1 = context.stock_str(StockMessage::SelfMsg).strdup();
             (*lot).text1_meaning = 3i32
         }
-    } else if chat.is_null() {
-        (*lot).text1 = 0 as *mut libc::c_char;
-        (*lot).text1_meaning = 0i32
-    } else if (*chat).type_0 == 120i32 || (*chat).type_0 == 130i32 {
+    } else if chat.typ == Chattype::Group || chat.typ == Chattype::VerifiedGroup {
         if 0 != dc_msg_is_info(msg) || contact.is_none() {
             (*lot).text1 = 0 as *mut libc::c_char;
             (*lot).text1_meaning = 0i32
         } else {
-            if !chat.is_null() && (*chat).id == 1i32 as libc::c_uint {
+            if chat.id == 1 {
```

Comment:
DC_CHAT_ID_DEADDROP (recommend to subtitute these numbers in your PR because it makes reviewing the functionality / logic easier)

Reply:
that will come in a later step, doing things commitwise so I get less complaints about touching too many things at once ;)